src/sentry/models/grouphashmetadata.py

```diff
@@ -4,18 +4,54 @@
 from sentry.backup.scopes import RelocationScope
 from sentry.db.models import Model, region_silo_model
 from sentry.db.models.base import sane_repr
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
 
 
 @region_silo_model
 class GroupHashMetadata(Model):
     __relocation_scope__ = RelocationScope.Excluded
 
     # GENERAL
+
     grouphash = models.OneToOneField(
         "sentry.GroupHash", related_name="_metadata", on_delete=models.CASCADE
     )
     date_added = models.DateTimeField(default=timezone.now)
 
+    # SEER
+
+    # Only one hash representing each group is sent to Seer. For the grouphash actually sent, this
+    # field and the `grouphash` field will be identical. For the grouphashes assigned to the same
+    # group but which aren't sent, this will point to the GroupHash record for the sent hash. Note
+    # that because of merging/unmerging, the sent GroupHash and this metadata's GroupHash (if not
+    # one and the same) aren't guaranteed to forever point to the same group (though they will when
+    # this field is written).
+    seer_grouphash_sent = FlexibleForeignKey(
```

Comment:
i would prefer not to keep track of this on `GroupHashMetadata`. -- we can simply infer this between the relations between the grouphash and group.

Reply:
i see note above on merging/unmerging, i'll think further on this

Reply:
Copying here for posterity the highlights of our convo in Slack:

The two situations in which there will be more than one grouphash are some combo of there being a config transition happening and there being separate full-stacktrace and in-app-only hashes. After some investigation/further thinking I realized (confirmed by Armen) that 

a) We thought it was a UI bug that grouping info sometimes shows a green check mark next to both in-app and system, but in fact it’s a logic bug - in-app should take precedence but they both get marked as contributing to grouping. Only contributing hashes make it through to the point where we’re calling Seer, so if we fix the precedence bug, there will only be one contributing hash (and therefore no extra grouphashes to link to the one sent to Seer).

b) The whole point of secondary hashes is to preserve the continuity of an existing group. If there is no existing group, the secondary has no purpose, and similarly therefore shouldn’t make it through to the potential Seer call.

So, TL;DR - I can indeed take that link out, because once we fix the above problems, there will only ever be one hash, meaning nothing to link.